packages/agents-hosting/src/app/guards/guardStorage.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ActiveGuard } from './types'
+import { TurnContext } from '../../turnContext'
+import { Storage } from '../../storage'
+
+/**
+ * Storage manager for guard state.
+ */
+export class GuardStorage {
+  constructor (private storage: Storage, private context: TurnContext) { }
```

Comment:
The constructor is missing documentation. Is it because of the private properties?